kratos/python/add_matrix_to_python.cpp

```diff
@@ -191,6 +191,11 @@ namespace Python
 
           return matrix;
         }));
+
+
+        py::implicitly_convertible<py::buffer, DenseMatrix<double>>();
```

Comment:
I am guessing this could cause the tests in the LinearSolversApp to fail
The rest seems unrelated

Reply:
this one is simply telling that you can pass a numpy matrix to a function that expects a DenseMatrix

it should work silently...(and it has nothing to do with the linear solvers app...)

Reply:
true, now I saw it also failing in other PRs